.eslintrc.js

```diff
@@ -78,6 +76,5 @@ module.exports = {
     'react/jsx-no-bind': 'off', // Keep perf implications in mind, but was giving too many warnings and hurting readability
     curly: 'error',
     radix: 'error',
-    'prettier/prettier': ['warn', prettierConfig],
```

Comment:
I found the linting squigglies to be distracting for issues that would be fixed on save or during the commit stage. I opted to remove from the default config and move to a separate config specifically for formatting. The `.eslintrc-prettier.js` config file is called for formatting purposes only.